### PR TITLE
[4.0] Replace double quote with single quote

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
For consistency, replace double quote with singe quote.

### Testing Instructions
Code review.

### CODE REVIEW

File: layouts/joomla/form/field/contenthistory.php [hunk 1]
```diff
@@ -61,7 +61,7 @@
 		'modalWidth'  => '80',
 		'bodyHeight'  => '60',
 		'footer' => '<button type="button" class="btn btn-secondary" data-bs-dismiss="modal" aria-hidden="true">'
-			. Text::_("JLIB_HTML_BEHAVIOR_CLOSE") . '</button>'
+			. Text::_('JLIB_HTML_BEHAVIOR_CLOSE') . '</button>'
```
Comment (brianteeman) [inline layouts/joomla/form/field/contenthistory.php:64]: pretty sure that wont work

**Comment by alikon** [comment]
what it is supposed to be standard? if there is 1

**Comment by chmst** [comment]
In this case it is inside of a single quote and breaks it. A variable wich concatenated strings would be better than nested strings.

File: layouts/joomla/form/field/contenthistory.php (hunk 1 above)
Comment (Bakual) [reply layouts/joomla/form/field/contenthistory.php:64]: Why shouldn't it?

**Comment by Bakual** [comment]
> what it is supposed to be standard? if there is 1

There is no official standard. 😄

File: layouts/joomla/form/field/contenthistory.php (hunk 1 above)
Comment (brianteeman) [reply layouts/joomla/form/field/contenthistory.php:64]: because its already inside a single quote isnt it?

**Comment by HLeithner** [comment]
Always use single quote except you want to use fast rope which is faster but personally I don't variables within a text block, it's less readable without a syntax highlighter.

File: layouts/joomla/form/field/contenthistory.php (hunk 1 above)
Comment (richard67) [reply layouts/joomla/form/field/contenthistory.php:64]: No. The previous row ends with a single quote and then the line changed here is appended to that with the dot at the beginning of the line.

Comment (brianteeman) [reply layouts/joomla/form/field/contenthistory.php:64]: grrh github preview :(

Comment (Bakual) [reply layouts/joomla/form/field/contenthistory.php:64]: Sometimes it's a bit tricky to read 😃

File: components/com_content/tmpl/article/default_links.php [hunk 2]
```diff
@@ -74,7 +74,7 @@
 									'modalWidth'  => '500',
 									'bodyHeight'  => '500',
 									'footer' => '<button type="button" class="btn btn-secondary" data-bs-dismiss="modal" aria-hidden="true">'
-										. \Joomla\CMS\Language\Text::_("JLIB_HTML_BEHAVIOR_CLOSE") . '</button>'
+										. \Joomla\CMS\Language\Text::_('JLIB_HTML_BEHAVIOR_CLOSE') . '</button>'
```
Comment (alikon) [inline components/com_content/tmpl/article/default_links.php:77]: why not 
`use Joomla\CMS\Language\Text;`
and 
`Text::_('JLIB_HTML_BEHAVIOR_CLOSE')`

**Review by alikon — CHANGES_REQUESTED** [review]

File: components/com_content/tmpl/article/default_links.php (hunk 2 above)
Comment (Bakual) [reply components/com_content/tmpl/article/default_links.php:77]: Because it's out of scope of this PR. Scope is "Replace double quote with single quote".

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on e299c59bd9cc59f4b7a1411543fd7582b379bdc4<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34452">issues.joomla.org/tracker/joomla-cms/34452</a>.</sub>

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on e299c59bd9cc59f4b7a1411543fd7582b379bdc4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34452">issues.joomla.org/tracker/joomla-cms/34452</a>.</sub>

**Comment by joomdonation** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34452">issues.joomla.org/tracker/joomla-cms/34452</a>.</sub>

**Comment by drmenzelit** [comment]
Thanks

**Pull request merged**